src/sentry/static/sentry/app/views/settings/organizationRelay/modals/add/step.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+
+type Props = {
+  title: React.ReactNode;
+  content: React.ReactElement;
+  subtitle?: React.ReactNode;
+  /**
+   * The position of the step.
+   * The prop defaults to the value inherited from the parent Steps component.
+   */
+  index?: number;
+};
+
+const Step = ({title, subtitle, content, index}: Props) => (
```

Comment:
I think @robinrendle wanted this to be a part of the List component
https://storybook.getsentry.net/branches/master/index.html?path=/story/core-list--default

Reply:
ok I was not aware of this component...thanks for pointing this out 🙂